docs/farming-&-staking/farming/pulsar/pulsar-tips.mdx

```diff
@@ -17,6 +17,36 @@ import TabItem from '@theme/TabItem';
 
 Welcome to the Additional Tips section! Whether you're a seasoned farmer or just starting out with the Subspace Network, these tips and tricks are designed to enhance your experience and efficiency. Here, we delve into practical advice and lesser-known techniques to help you fine-tune your farming setup and navigate common challenges with ease. From configuring your environment to managing background processes, these insights are tailored to ensure your Farmer operates smoothly and effectively. Let's dive in and explore how you can get the most out of your Subspace Network Farmer.
 
+### Switching to a new release
+
+:::info
+Unless specifically mentioned by the Development team you should **NOT** have to wipe your node and configuration on new releases.
+:::
+
+In general you should be able to download the latest release of **Pulsar**, and re-start with the same commands as you started to prior version with no errors.
+
+There are some cases where version updates will cause issue with **Pulsar** and you may have to use the `./pulsar_release wipe` command, typically when errors occur. 
+If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
+
+### Wipe Node & Farmer
+
+:::tip Updated from a previous version and now having issues?
+
+Occasionally after updating to a new version of the Pulsar you will need to wipe your node and farmer, generally this should not be required but can be attempted if your farmer is having issues after having had worked fine previously.
```

Comment:
```suggestion
Occasionally after updating to a new version of Pulsar you will need to wipe your node and farmer, generally this should not be required but it can be attempted if your farmer is having issues after having been working fine previously. Particularly if there is suspected data corruption e.g. after a power outage.
```